hack/README.md

```diff
@@ -47,15 +47,16 @@ export REPO_PREFIX=gcr.io/google-samples/microservices-demo
 
 #### 8. Once your pull-request is approved, merge it.
 
-#### 9. Connect to our [onlineboutique-master GKE cluster](https://pantheon.corp.google.com/kubernetes/clusters/details/us-central1-c/online-boutique-master/details?project=onlineboutique-ci)
+#### 9. Connect to our [online-boutique-release GKE cluster](https://pantheon.corp.google.com/kubernetes/clusters/details/us-central1-c/online-boutique-release/details?project=online-boutique-ci)
```

Comment:
I named the new GKE cluster used for [onlineboutique.dev](https://onlineboutique.dev) "online-boutique-release".